crates/oci-distribution/src/lib.rs

```diff
@@ -188,6 +222,43 @@ impl Client {
         headers
     }
 }
+#[async_trait]
+pub trait ImageStore {
```

Comment:
I think in the CRI experiment we decided to call this a `ModuleStore`, as it is more accurate as to what it stores.